addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1599,6 +1599,106 @@ QUnit.test('[technical] chat window with a thread: keep scroll position in messa
     );
 });
 
+QUnit.test('chat window does not fetch messages if hidden', async function (assert) {
+    assert.expect(14);
+
+    // 3 channels are expected to be found in the messaging menu, each with a
+    // random unique id that will be referenced in the test
+    this.data['mail.channel'].records = [
+        {
+            id: 1,
+            is_minimized: true,
+            name: "Channel #1",
+            state: 'open',
+        },
+        {
+            id: 2,
+            is_minimized: true,
+            name: "Channel #2",
+            state: 'open',
+        },
+        {
+            id: 3,
+            is_minimized: true,
+            name: "Channel #3",
+            state: 'open',
+        },
+    ];
+    await this.start({
+        env: {
+            browser: {
+                innerWidth: 900, // enough to fit 2 chat windows but not 3
```

Comment:
Prefer providing the detailed calculus for this value, like some other chat window manager tests.